src/test/regress/specs/isolation_add_node_vs_copy.spec

```diff
@@ -0,0 +1,71 @@
+setup
+{
+	truncate pg_dist_shard_placement;
+	truncate pg_dist_shard;
+	truncate pg_dist_partition;
+	truncate pg_dist_colocation;
+	truncate pg_dist_node;
```

Comment:
It seems better to handle this by cleaning up in the `teardown` section, which seems to already happen?